### DISCUSSION TITLE
`next` route style doesn't seem to support `Optional catch all routes`

### DISCUSSION THREAD

**Discussion by jiangmaniu** [hannoeru/vite-plugin-pages#486] [Ideas]
### Describe the bug

I tried using `next` style [Optional catch all routes](https://nextjs.org/docs/routing/dynamic-routes#optional-catch-all-routes) but it doesn't seem to work

![image](https://user-images.githubusercontent.com/50100681/190352149-749aa77d-a084-483c-b37a-bafe03cd1b89.png)

![image](https://user-images.githubusercontent.com/50100681/190352490-f7e63ccf-d860-4823-895b-0eaacf53e77c.png)

### expected goal

Can match both `/redirect` and `/redirect/:all(.*)`

### Reproduction

x

### System Info

```shell
x
```

### Used Package Manager

pnpm

### Validations

- [X] Follow the [Code of Conduct](https://github.com/hannoeru/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the docs.
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A please open a GitHub Discussion.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by hannoeru** [comment]
This feature is currently not supported.
